message/validation/message_counts.go

```diff
@@ -108,13 +100,8 @@ func (c *MessageCounts) RecordConsensusMessage(signedSSVMessage *spectypes.Signe
 	case specqbft.PrepareMsgType:
 		c.Prepare++
 	case specqbft.CommitMsgType:
-		switch {
-		case len(signedSSVMessage.GetOperatorIDs()) == 1:
+		if len(signedSSVMessage.GetOperatorIDs()) == 1 {
 			c.Commit++
-		case len(signedSSVMessage.GetOperatorIDs()) > 1:
-			c.Decided++
-		default:
-			panic("unexpected signers length 0") // 0 length should be checked before
 		}
```

Comment:
what happens if there are more than 1 signers? does this mean you can keep sending messages with more than 1 signers as an attack to avoid being blocked by the counter?

Reply:
We don't need to check the message count for decided messages here as it would be unreachable because we already check that the number of signers is correct and the same signers were not seen